mmrotate/datasets/pipelines/transforms.py

```diff
@@ -80,8 +80,10 @@ def bbox_flip(self, bboxes, img_shape, direction):
         flipped = bboxes.copy()
         if direction == 'horizontal':
             flipped[:, 0] = img_shape[1] - bboxes[:, 0] - 1
+            flipped[:4] = flipped[[1, 0, 3, 2]].copy()
```

Comment:
This new line of code will produce incorrect image flips, especially for the original centered representation. Please check it and fix it.